### ISSUE TITLE
Alacritty support?

### ISSUE THREAD

**Issue by agoodfellow123** [cosmos72/twin#94]
Hello, I can briefly remember twin with mouse support working fine on Alacritty before. However in my new setup I have realized that twin on Alacritty with mouse support doesn't work. I'm on Arch so this might be a new issue Alacritty just introduced,

Here is the usual error report:
```
twin: starting display driver module `hw_tty'...
      linux_InitVideo() failed: terminal `alacritty' is not `linux'.
      GPM_InitMouse() failed: terminal `/dev/pts/25'
      is not a local linux console.
      xterm_InitMouse() failed: terminal `alacritty' is not supported.

        ALL  MOUSE  DRIVERS  FAILED.
```

From the error, I can see that Alacritty is not supported both for mouse and graphics (Although it still works without mouse but looks abismal) and twin is looking for an xterm. I'm assuming my memory is misleading me, regardless support for Alacritty would be much appreciated.

**Comment by cosmos72** [comment]
Hello @agoodfellow123,
as far as I remember, there's no standard mechanism to programmatically detect whether a terminal supports mouse or not.

Currently, twin checks if the environment variable `$TERM` is either `linux`, `xterm` or some known xterm-like variant (`rxvt` or `Eterm`).

You can easily add `alacritty` to the checks in `twin/server/hw/hw_tty_common/mouse_xterm.h` line 54 and recompile,
although a more general solution would surely be better...

Another "quick and dirty" approach that works with many terminal emulators (provided they are similar enough to xterm) is to just set `TERM=xterm` before starting twin.

Let me know if either solution works :)

**Comment by agoodfellow123** [comment]
I am happy to report that "quick and dirty" method works perfectly! Haven't tried the compiling method but since lying about the terminal works I assume it would work as well.

**Comment by cosmos72** [comment]
Good :) then a slightly more long-term solution is to launch twin with
```
twstart --hw=tty,mouse=xterm
```
which avoids lying about `$TERM` environment variable.
